src/contracts/mixins/GPv2Signing.sol

```diff
@@ -35,6 +35,10 @@ abstract contract GPv2Signing {
     /// @dev The EIP-712 domain version used for computing the domain separator.
     bytes32 private constant DOMAIN_VERSION = keccak256("v2");
 
+    /// @dev Marker value indicating an order is pre-signed.
+    uint256 private constant PRE_SIGNED =
+        uint256(keccak256("GPv2Signing.Scheme.PreSign"));
```

Comment:
<a href="https://gitme.me/image?url=https%3A%2F%2Fmedia1.giphy.com%2Fmedia%2F134iGJcqa8A9lC%2Fgiphy.gif&token=fancy" data-gitmeme-token="fancy"><img src="https://media1.giphy.com/media/134iGJcqa8A9lC/giphy.gif" title="Created by gitme.me with /fancy"/></a>

Reply:
This was only to make both `filledAmount` and `preSignature` fields `mapping(bytes => uint256)` so that we could reuse the `freeOrderStorage` :stuck_out_tongue: